prdoc/schema_user.json

```diff
@@ -0,0 +1,228 @@
+        "type": "array",
+        "description": "Description of the PR. Provide a description for each relevant audience.\nSee the `audience` property for more documentation about audiences",
+        "items": {
+          "$ref": "#/$defs/doc"
+        },
+        "minItems": 1
+      },
+      "crates": {
+        "description": "You have the option to provide input about the semver level of your changes for each crate\n. This is used during the crate publishing to crates.io and to help users understand the impact of the changes introduced in your PR.\nYou can also set it to `no change` if you know there was no significant semver change",
+        "type": "array",
+        "items": {
+          "$ref": "#/$defs/crate"
+        }
+      },
+      "migrations": {
+        "description": "It is important for users to be aware of migrations.\nMake sure to mention any migrations in the appropriate sub-properties",
+        "type": "object",
+        "properties": {
+          "db": {
+            "type": "array",
+            "nullable": false,
+            "description": "List of the Database Migrations or []",
+            "items": {
+              "$ref": "#/$defs/migration_db"
+            },
+            "minItems": 0,
+            "required": [
+              "name",
+              "description"
+            ]
+          },
+          "runtime": {
+            "type": "array",
+            "nullable": false,
+            "description": "List of the Runtime Migrations or []",
+            "minItems": 0,
+            "items": {
+              "$ref": "#/$defs/migration_runtime"
+            },
+            "required": [
+              "db",
+              "runtime"
+            ]
+          }
+
+        }
+      },
+      "host_functions": {
+        "description": "Lits of the host functions involved in this PR.\nThis is used to generate the host functions documentation.",
+        "type": "array",
+        "minItems": 0,
+        "items": {
+          "$ref": "#/$defs/host_function"
+        }
+      }
+    },
+    "required": [
+      "title",
+      "doc",
+      "crates",
+      "migrations",
+      "host_functions"
+    ],
+    "$defs": {
+      "audience": {
+        "oneOf": [
+          {"const": "Node Dev", "title": "Someone working on the node"},
+          {"const": "Runtime Dev", "title": "Someone working on runtimes, frames, pallets"},
+
+          {"const": "Node Operator", "title": "Someone running a node in general, not specific to validators"},
+          {"const": "Validator", "title": "Someone running a Validator node"},
+
+          {"const": "Parachain Builder", "title": "Someone building a parachain"},
+
+          {"const": "App Builder", "title": "Someone building Apps and dApps on the shoulder of the Polkadot network and its parachains"},
+
+          {"const": "Runtime User", "title": "Someone involved or interested in the governance process or noticeable changes to the runtime and/or its dependencies"}
```

Comment:
Please update to the audience as discussed in the forum.

Reply:
Fixed in https://github.com/paritytech/polkadot-sdk/pull/1946/commits/83982d5d8aa3d956d0264ef2a025c8774d42abf4